src/Stratis.Bitcoin.Features.Dns.Tests/GivenADnsFeature.cs

```diff
@@ -31,8 +31,8 @@ public void WhenConstructorCalled_AndDnsServerIsNull_ThenArgumentNullExceptionIs
             IWhitelistManager whitelistManager = new Mock<IWhitelistManager>().Object;
             ILoggerFactory loggerFactory = new Mock<ILoggerFactory>().Object;
             INodeLifetime nodeLifetime = new Mock<INodeLifetime>().Object;
-            NodeSettings nodeSettings = NodeSettings.Default();
-            DnsSettings dnsSettings = new Mock<DnsSettings>().Object;
+            NodeSettings nodeSettings = NodeSettings.Default(KnownNetworks.TestNet);
```

Comment:
Shouldn't this be Mainnet? Same for the other changes in the class.

Reply:
As it was before yes, but we should probably try to use testnet for tests as a general approach, in this case the tests pass and the distinction between main/test does not matter.